edge/pkg/metamanager/metaserver/kubernetes/scope/scope.go

```diff
@@ -46,26 +48,32 @@ func NewRequestScope() *handlers.RequestScope {
 
 		Serializer:     serializer.NewNegotiatedSerializer(),
 		ParameterCodec: scheme.ParameterCodec,
-		//Creater:         nil,
-		Convertor: fakers.NewFakeObjectConvertor(),
-		Defaulter: fakers.NewFakeObjectDefaulter(),
-		Typer:     crdserverscheme.NewUnstructuredObjectTyper(),
-		//UnsafeConvertor: nil,
-		Authorizer: fakers.NewAlwaysAllowAuthorizer(),
+
+		StandardSerializers: make([]runtime.SerializerInfo, 0),
+
+		Creater:         unstructuredscheme.NewUnstructuredCreator(),
```

Comment:
Perhaps we could try dividing it into three dimensions to generate creator, kubernetes, kubeedge, and other resources.

Reply:
When and how should implemented creaters for CRDs to be injected or registered into it? 

I'm confused about how the creaters will work. Should CRDs be synchronized from `cloudcore` and then `metaserver` update the creaters for them according to the definitions?

Reply:
And what about convertor and typer? Do they need to be divided also?

Reply:
@ZhengXinwei-F Hi, any further idea? Can we merge the fix to the panic first anyway?

Reply:
@fisherxu Hi, could you please have a look at this PR, can we merge the fix to the panic first anyway?